### ISSUE TITLE
Fail to deploy a .NET project to a Windows .NET Web App

### ISSUE THREAD

**Issue by v-xinda** [microsoft/vscode-azureappservice#2279]
**OS:** M1(Pro)
**Build Version:** 20220630.3
**Regression:** Previous release

**Repro Steps:**
1. Create a .NET project using command "dotnet new web".
2. Create a Windows .NET Web App.
3. Deploy the .NET project to the Windows .NET Web App.
4. Check whether succeeds to deploy a .NET project to the Windows .NET Web App.

**Expect:**
Succeed to deploy a .NET project to the Windows .NET Web App.

**Actual:**
Fail to deploy a .NET project to the Windows .NET Web App.
![image](https://user-images.githubusercontent.com/57662131/176625345-36101931-03a9-475e-b852-796109389f69.png)
Output Log:
```
2:54:08 PM nicole-net6-win: Starting deployment...
2:54:22 PM nicole-net6-win: Creating zip package...
2:54:23 PM nicole-net6-win: Zip package size: 34.5 kB
2:54:33 PM: Error: The request was aborted
```

**More Info:**
1. This issue reproduces on Windows and Linux unstably.
2. This issue also reproduces when deploying a Nodejs project to the Nodejs web app and it is unstable.
3. This issue doesn't reproduce when deploying a .NET project to the Linux/Basic .NET Web App.

**Comment by nturinski** [comment]
Both @alexweininger and I couldn't reproduce this after following the repro steps.  I tried to deploy like 5 different occasions on new apps, old apps, deleted and re-created apps, and two deployments running simultaneously.

The code that was changed just adds an optional GET request.  This shouldn't be throwing a "The request was aborted" error, so I feel like that this error may just be a service issue.  It doesn't seem likely to me that this is a regression.
